src/bitcoin-cli.cpp

```diff
@@ -642,7 +642,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
         "  send     Time since last message sent to the peer, in seconds\n"
         "  recv     Time since last message received from the peer, in seconds\n"
         "  txn      Time since last novel transaction received from the peer and accepted into our mempool, in minutes\n"
-        "           \"*\" - whether we relay transactions to this peer (relaytxes is false)\n"
+        "           \"*\" - we do not relay transactions to this peer (relaytxes is false or not available)\n"
```

Comment:
if it is not available it will default to true, no?

Reply:
@MarcoFalke 
Yes, I was able to verify your point

On line https://github.com/bitcoin/bitcoin/blob/master/src/bitcoin-cli.cpp#L542, the `"*"` or `""` is selected based on bool value of `is_tx_relay`.

```cpp
peer.last_trxn ? ToString((time_now - peer.last_trxn) / 60) : peer.is_tx_relay ? "" : "*",
```

and,
on line https://github.com/bitcoin/bitcoin/blob/master/src/bitcoin-cli.cpp#L481, `is_tx_relay` defaults to **True** when `relaytxes` is **Null**

```cpp
const bool is_tx_relay{peer["relaytxes"].isNull() ? true : peer["relaytxes"].get_bool()};
```

Reply:
Thanks, done, and added displaying the new "not available" state distinctly from the "true but no last tx time" state.